src/app/models/Region.ts

```diff
@@ -0,0 +1,21 @@
+export interface Region {
+  name: string;
+  code: string;
+  latitude: number;
+  longitude: number;
+}
+
+export const regions: Region[] = [
```

Comment:
We could use a package with country info which comes with all this stuff, and tack on a `global` entry to their list. 

Most apis I've looked at prefer the 2 letter country code over using the country name to search too.

I've used the `iso-3166-1` package in #19 but that doesn't have latitude and longitude data. So might need to change over to something else if thats required.

Reply:
I found this
https://www.npmjs.com/package/country-finder
but I'm inclined to leave it for another PR

Reply:
looks good and yeah, new PR for it would make sense.
Its also got continent grouping which could be useful later